feature-group-definitions/transforms2d.yml

```
@@ -0,0 +1,19 @@
+name: CSS 2D transforms
```

Given our recent changes to [the guidelines](https://github.com/web-platform-dx/web-features/tree/main/docs#feature-guidelines), what do you think of this approach instead?

```suggestion
name: 2D transforms (CSS)
```

---

Maybe we just put CSS in the description?

---

Yes, that would be good too. I almost suggested it, but I figured that mean signing myself up to write them… which looked hard. 😅